backend/main/chapters/c06_lists.py

```diff
@@ -829,16 +826,367 @@ class sum_list(Step):
 - **`subscripting`**: Get a value at an index. `nums[0]` is 1, `nums[1]` is 2, `nums[2]` is 3.
 - **`+`**: Concatenates lists. `nums + [4, 5]` is `[1, 2, 3, 4, 5]`.
 
-Here's some new things. Try them out in the shell.
+Note that `nums.append(4)` modifies the existing list `nums`, while `nums + [4, 5]` does not.
+To preserve the value of `nums + [4, 5]` we must assign it to a *new variable*.
+Type and run the following code in the editor:
+
+__program_indented__
```

Comment:
```suggestion
Run the following code:

    __copyable__
    __program_indented__
```

Typing in this code doesn't provide much learning benefit. It's tedious and all the syntax is familiar.

Reply:
OK.